components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -214,6 +257,24 @@ export function CreateWorkspacePage() {
                         {isLoading ? "Loading ..." : isStarting ? "Creating Workspace ..." : "New Workspace"}
                     </Button>
                 </div>
+                {existingWorkspaces.length > 0 && (
+                    <div className="w-full flex flex-col justify-end px-6">
+                        <p className="mt-6 text-center text-base">Running workspaces on this revision</p>
```

Comment:
**question**: If it's not too much trouble and we want to add this in the scope of this PR, it could be helpful to be specific about the number of similar workspaces.

```
// BEFORE
Running workspaces on this branch

// AFTER
3 running workspaces on this branch
```